developer-docs/docker.md

```diff
@@ -65,7 +60,7 @@ And then generate the certificate again using the `ca generate` command from abo
 
 [Postman](https://www.getpostman.com/) is advisable to verify that the endpoints are configured. In order to set up Postman, navigate to Settings > Certificates and add client certificates for hosts `0.0.0.0:8140` and `0.0.0.0:44633` where the CRT file points to `spec/volumes/puppet/ssl/certs/aceserver.pem` and Key file points to `spec/volumes/puppet/ssl/private_keys/aceserver.pem`
 
-*Note*: These cert and key files will only be created when the PuppetServer container has finished initalising.
+*Note*: These cert and key files will only be created when the PuppetServer container has finished initalising and the `ca generate` command has been used.
```

Comment:
is that something the user needs to do (as implied by "command has been used") or is that something that happens during startup?

Reply:
The users are asked to execute the command in the setup steps of the Puppetserver, https://github.com/puppetlabs/ace/pull/42/files#diff-25a45b3d9df8e2e8cc48099f9b67f1c7R32